docs/how-to-debug-a-hyperlight-guest.md

```diff
@@ -0,0 +1,111 @@
+# How to debug a Hyperlight guest using gdb
+
+Hyperlight supports gdb debugging of a guest running inside a Hyperlight sandbox.
+When Hyperlight is compiled with the `gdb` feature enabled, a Hyperlight sandbox can be configured
+to start listening for a gdb connection.
+
+## Supported features
+
+The Hyperlight `gdb` feature enables:
+
+1. KVM guest debugging:
```

Comment:
Should we call out more clearly that this only works on KVM for now?

Reply:
It might be better to advertise it as only for KVM for now